basictracer/scope.py

```diff
@@ -23,33 +23,28 @@
 from opentracing import Scope
 
 
-class BasicScope(Scope):
+class ThreadLocalScope(Scope):
     """BasicScope is the implementation of `opentracing.Scope`"""
 
-    def __init__(self, manager, span, finish_on_close=True):
+    def __init__(self, manager, span, finish_on_close):
```

Comment:
since developers are not going to use it directly (but only via the `Tracer` API), I'm in favor of removing the kwarg making it an arg